diff --git a/dist/utils/document/load.js b/dist/utils/document/load.js
index 8fa1a94fcac5409483b05b4091781d8ff9d833a8..725337ae3d97bcd1b15a9ed65d2d92c8b00b60d8 100644
--- a/dist/utils/document/load.js
+++ b/dist/utils/document/load.js
@@ -1,4 +1,4 @@
-import { upgrade } from "../../node_modules/.pnpm/@scalar_openapi-upgrader@0.2.9/node_modules/@scalar/openapi-upgrader/dist/upgrade.js";
+import { upgrade } from "@scalar/openapi-upgrader";
 import { bundle } from "@fumadocs/api-docs/schema/bundle";
 //#region src/utils/document/load.ts
 /**
